added release notes for 10.14.0.370 (#227)
SoftwareAG/c8y-releasenotes: content/release-10-14-0/devices-10-14-0-bundle/10-14-0-370.md

---
weight: 16
title: Release 10.14.0.370
layout: bundle
---

<!--10.14.0.349 - 10.14.0.370-->

### Fixes

<div><table ><colgroup>
<col style="width: 15%;"><col style="width: 70%;"><col style="width: 15%;"></colgroup>
<thead><tr>
<th>
Component</th>
<th>
Description</th>
<th>
Issue</th>
</tr>
</thead><tbody>

<tr>
<td>Device Management</td>
<td>In the SmartREST template editor, issues with the presentation of the <b>External ID type</b> field for inventory POST messages have been fixed. Under <b>CSV preview</b>, the generated "Template creation CSV" has been adjusted to include the "ID", "externalId" and "externalIdType" values.</td>
<td>DM-2093</td>
</tr>

<tr>
<td>Device Management</td>
<td>In case of bad connectivity or network delay gateway devices could go to a state where they were disconnected. This resulted in operation execution being suspended. This issue is now resolved.</td>
<td>DM-2037</td>
</tr>

<tr>
<td>Device Management</td>
<td>In the device grid, issues with applying filters have been fixed.</td>
<td>DM-2321</td>
</tr>

<tr>
<td>OPC UA</td>

> <td>If the namespace is not defined for a specific node, a missing error handler in the past prevented completion of the scan operation. This is now fixed by skipping nodes that cannot be read. The opcua-device-gateway logs include further details in such cases. The log level INFO contains the failure message and the log level DEBUG contains the stacktrace.</td>

Reviewer: ```suggestion
<td>If the namespace was not defined for a specific node, a missing error handler prevented completion of the scan operation. This is now fixed and nodes which cannot be read are skipped. The opcua-device-gateway logs include further details in such cases. The log level INFO contains the failure message and the log level DEBUG contains the stacktrace.</td>
```
